zingolib/src/wallet/transaction_record.rs

```diff
@@ -717,18 +743,31 @@ mod tests {
         );
     }
 
-    #[test]
-    fn total_value_received() {
-        let transaction_record = nine_note_transaction_record();
-        let old_total = transaction_record
-            .pool_value_received::<orchard::note_encryption::OrchardDomain>()
-            + transaction_record
-                .pool_value_received::<sapling_crypto::note_encryption::SaplingDomain>()
-            + transaction_record
-                .transparent_outputs
-                .iter()
-                .map(|utxo| utxo.value)
-                .sum::<u64>();
-        assert_eq!(transaction_record.total_value_received(), old_total);
+    proptest! {
+        #[test]
+        #[allow(clippy::too_many_arguments)]
+        fn total_value_received(
+            transparent_unspent: u32,
```

Comment:
and `u32` here?

Reply:
because proptest wants to add nine of these to get 1 of those